Smart Village App/.idea/smart-village-theme.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
should this file be named differently according to the new folder name?

Reply:
Hi @donni106 
We commit a push again where we rename this file also from smart-village-theme to Smart Village app